src/provider/bpmn/properties/AdHocCompletionProps.js

```diff
@@ -0,0 +1,159 @@
+import { is, getBusinessObject } from 'bpmn-js/lib/util/ModelUtil';
+
+import {
+  isFeelEntryEdited,
+  CheckboxEntry,
+  isCheckboxEntryEdited,
+} from '@bpmn-io/properties-panel';
+
+import { createElement } from '../../../utils/ElementUtil';
+
+import { useService } from '../../../hooks';
+
+import { FeelEntryWithVariableContext } from '../../../entries/FeelEntryWithContext';
+
+/**
+ * @typedef { import('@bpmn-io/properties-panel').EntryDefinition } Entry
+ */
+
+/**
+ * @returns {Array<Entry>} entries
+ */
+export function AdHocCompletionProps(props) {
```

Comment:
I was first inclined to call this `CompletionProps`, but as these properties are only defined for ad hoc subprocesses I decided to go with this naming. Happy to adapt to a more generic naming to follow the pattern of the other props if needed.

Reply:
I think it's OK to have a file dedicated for ad-hoc subprocess. There's another place where we set the same property, but it's in a different context, so I'd rather have some duplication with adjustments possible than non-adjustable, shared entry.

![image](https://github.com/user-attachments/assets/25d64dc1-3dc8-4ae4-b4d2-4ae57221072b)